app/views/admin/users/_member_index.html.erb

```diff
@@ -7,34 +7,64 @@
 
     <!-- Small screen data view start -->
     <div class="block m:hidden fs-s">
-      <h2 class="member-data-heading fs-s py-2 -mx-4 px-6 color-base-60">Members</h2>
-      <ul class="list-none mx-2">
-        <% @users.each do |user| %>
+      <h2 class="member-data-heading fs-s py-2 -mx-3 px-3 color-base-60">Members</h2>
+      <ul class="list-none">
+        <% @users.includes([:organization_memberships]).each do |user| %>
           <li class="py-4">
             <article>
               <header class="flex">
-                <img class="radius-full mr-2" src="<%= user.profile_image_url_for(length: 50) %>" width="40" height="40" alt="" />
-                <div class="flex flex-col">
-                  <h3 class="fs-base"><%= link_to user.name, admin_user_path(user) %></h3>
-                  <span class="color-base-60">@<%= user.username %></span>
+                <a href="<%= admin_user_path(user.id) %>">
+                  <img class="radius-full align-middle" src="<%= user.profile_image_url_for(length: 50) %>" width="40" height="40" alt="<%= user.name %>" />
+                </a>
+                <div class="pl-3 flex-1 fs-s"><%= link_to user.name, admin_user_path(user) %>
+                  <p class="color-base-60">@<%= link_to user.username, admin_user_path(user) %>&nbsp;
+                  </p>
                 </div>
               </header>
               <div class="flex justify-between py-2">
-                <div>standing info</div>
-                <div>role info</div>
+                <div>
+                <% if user.suspended? %>
+                  <span><%= inline_svg_tag("suspended-circle.svg", aria: true, title: "Suspended", class: "mb-1") %>Suspended</span>
+                <% elsif user.warned? %>
+                  <span><%= inline_svg_tag("warned-circle.svg", aria: true, title: "Warned", class: "mb-1") %>Warned</span>
+                <% elsif user.comment_suspended? %>
+                  <span><%= inline_svg_tag("suspended-circle.svg", aria: true, title: "Comment Suspended", class: "mb-1") %>Comment suspended</span>
+                <% elsif user.trusted? %>
+                  <span><%= inline_svg_tag("trusted-circle.svg", aria: true, title: "Trusted", class: "mb-1") %>Trusted</span>
+                <% else %>
+                  <span><%= inline_svg_tag("good-standing-circle.svg", aria: true, title: "Good Standing", class: "mb-1") %>Good standing</span>
+                <% end %>
+                </div>
+                <div>
+                <span class="c-indicator ml-2 crayons-hover-tooltip" data-tooltip="<%= user.current_role %>">
+                  <%= user.current_role %>
+                </span>
```

Comment:
Thanks for the assistance with them! I noticed that as well, however, I think it'd be best to leave it for a later iteration, as it may be a bit more complex, as we'll have to determine which of the resource admin roles a member has via a conditional or a loop (this is my first thought at least--I tinkered with this a bit and there wasn't a great way to get that info super easily).

Reply:
I'll make note of this and add it to the issue for this work!